explorer/lib/explorer_web/live/utils.ex

```diff
@@ -128,10 +128,17 @@ defmodule ExplorerWeb.Helpers do
     Utils.binary_to_hex_string(binary)
   end
 
+  def is_stale?(batch) do
+    batch_ttl_secs = 300 # 5 minutes
+    expiration = Time.add(batch.submission_timestamp, batch_ttl_secs)
+    Time.after?(Time.utc_now(), expiration)
```

Comment:
This could also be done like this:
```suggestion
    Time.add(batch.submission_timestamp, 5, :minute)
    |> Time.before?(Time.utc_now())
```

Reply:
But, also, I think I should be using `DateTime`, will fix in the morning. `Time.add` discards the date so this would break overnight.